Function.cs

```diff
@@ -17,6 +17,14 @@ public class Function
 
         private readonly string[] coolColors =
         {
+			"Air Force Blue",
+			"American rose",
+			"Anroid Green",

```

Comment:
Looks like you're just missing the "d" in "Android".

```suggestion
			"Android Green",
```